hw7/iterator_fibo.py

```diff
@@ -0,0 +1,45 @@
+class Fibo:
+    def __init__(self, fibo_max=0):
+        self.a = 0
+        self.b = 1
+        self.c = 0
+        self.fibo_max = fibo_max
```

Comment:
Since fibo numbers increase rapidly with every next value I would suggest using here index instead of value. 
For example, fibo element at position 1000 has 209 digits and it would be very hard to set such a limit

Reply:
fixed